packages/server-render/server-register.js

```diff
@@ -1,9 +1,11 @@
 import { WebAppInternals } from "meteor/webapp";
-import { SAXParser } from "parse5";
 import MagicString from "magic-string";
-import { ServerSink } from "./server-sink.js";
+import { SAXParser } from "parse5";
+import streamer from "combined-stream2";
```

Comment:
Would `import { create as createStream } from "combined-stream2" work here?

Reply:
I'll try and see if that works!